dev/com.ibm.ws.org.apache.cxf.ws.security.2.6.2/bnd.bnd

```diff
@@ -0,0 +1,39 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-rt-ws-security;2.6.2;EXACT}}!/META-INF/MANIFEST.MF,bnd.overrides
+
+globalize: false
+instrument.ffdc: true
+
+cxfVersion=2.6.2
+
+-includeresource: \
+    @${repo;org.apache.cxf:cxf-rt-ws-security;${cxfVersion};EXACT}!/!META-INF/maven/*|\
+    META-INF/cxf/bus-extensions.txt=override/META-INF/cxf/bus-extensions.txt, \
+    cxf-ehcache.xml=resources/cxf-ehcache.xml, \
+    org/apache/cxf/ws/security/policy/interceptors/Messages.properties=resources/org/apache/cxf/ws/security/policy/interceptors/Messages.nlsprops, \
+    org/apache/cxf/ws/security/policy/model/Messages.properties=resources/org/apache/cxf/ws/security/policy/model/Messages.nlsprops, \
+    org/apache/cxf/ws/security/trust/Messages.properties=resources/org/apache/cxf/ws/security/trust/Messages.nlsprops, \
+    org/apache/cxf/ws/security/wss4j/Messages.properties=resources/org/apache/cxf/ws/security/wss4j/Messages.nlsprops
+
```

Comment:
It doesn't look like your including the Open-Liberty CXF source once it's compiled. The last line of  -includeresources header should be org/apache/cxf=${bin}/org/apache/cxf